File: Dandelion.java
import javax.swing.*;

public class Dandelion extends Plant{

    Dandelion(World world1, int x, int y) {
        SetStrength(0);
        SetInitiative(0);
        SetCords(x, y);
        SetMove(true);
        world = world1;
    }
    @Override
    public String getName() {
        return "DANDELION";
    }
    @Override
    public void Drawing() {
        ImageIcon icon = new ImageIcon("src/Icons/dandelion.png");
        world.GetButton(GetCords()).setIcon(icon);
    }
    @Override
    public void Action() {
        for (int i = 0; i < 3; i++) {
            //EMPTY PLACE
            int child = EmptyPLace();
            Polozenie tmp_cords = GetCords();
            if (child != -1) {

                if (child == Stale.LEFT) {
                        New(getName(), world, tmp_cords.x - 1, tmp_cords.y);
                }
                else if (child == Stale.RIGHT) {
                    New(getName(), world, tmp_cords.x + 1, tmp_cords.y);
                }
                else if (child == Stale.UP) {
                    New(getName(), world, tmp_cords.x, tmp_cords.y - 1);
                }
                else if (child == Stale.DOWN) {
                    New(getName(), world, tmp_cords.x, tmp_cords.y + 1);
                }
            }
        }
    }
}

File: dandelion.py
from plant import Plant
from Stale import Stale

class Dandelion(Plant):
    def __init__(self, world1, x, y):
        super().__init__()
        self.SetStrength(0)
        self.SetInitiative(0)
        self.SetCords(x, y)
        self.SetMove(True)
        self.world = world1

    def getName(self):
        return "DANDELION"

    def Drawing(self):
        self.world.buttons[self.GetCords().y -1][self.GetCords().x -1].configure(bg='#FFFF00')
    
    def New(self, text, reference, x, y):
        self.world.GetOrganisms().add(Dandelion(reference, x, y))    
    def Action(self):
        for i in range(3):
            # EMPTY PLACE
            child = self.EmptyPlace()
            tmp_cords = self.GetCords()
            if child != -1:
                if child == Stale.LEFT:
                    self.New(self.getName(), self.world, tmp_cords.x - 1, tmp_cords.y)
                elif child == Stale.RIGHT:
                    self.New(self.getName(), self.world, tmp_cords.x + 1, tmp_cords.y)
                elif child == Stale.UP:
                    self.New(self.getName(), self.world, tmp_cords.x, tmp_cords.y - 1)
                elif child == Stale.DOWN:
                    self.New(self.getName(), self.world, tmp_cords.x, tmp_cords.y + 1)
